src/templates/users/user_detail.html

```diff
@@ -25,13 +25,21 @@ <h2>{{ object.name }}</h2>
         </div>
       </div>
       <!-- End Action buttons -->
+      {% if friends_table %}
```

Comment:
all of this if block is inside another   {% if object == request.user %} and I don't see any object passed into the context

Reply:
This is because we only want the list of friends to show up when `object == request.user`. The list of friends should not show up when `object == request.admin`.